<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QFrame" name="ToolBar">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(179, 210, 230, 255), stop:1 rgba(240, 240, 240, 255));
border-color: rgb(188, 188, 188);
border: 0px solid;
margin: 0; 
padding: 0
;}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <widget class="QLabel" name="titleText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(232, 232, 232, 0);
border: 0px solid #ffffff;</string>
   </property>
   <property name="text">
    <string>CourseWare</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="pasteButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>2</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>paste</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
opacity: 0.5;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/paste.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="cutButton">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>3</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>cut</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>3</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>copy</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="textButton">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>6</y>
     <width>64</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZXiYuan-M01S</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>text block</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string>文本</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/text.png</normaloff>:/images/text.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="tableButton">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>3</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZXiYuan-M01S</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>table</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/table.png</normaloff>:/images/table.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="shapeButton">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>3</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZXiYuan-M01S</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>shape</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/shape.png</normaloff>:/images/shape.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="Line" name="line1">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>3</y>
     <width>3</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line2">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>3</y>
     <width>3</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="selButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>5</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/select.png</normaloff>:/images/select.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
